src/olympia/amo/tests/test_utils.py

```diff
@@ -258,12 +258,6 @@ def test_get_locale_from_lang(lang):
         assert locale.territory == territory
 
 
-@pytest.mark.parametrize('lang', settings.LANGUAGES_BIDI)
-def test_bidi_language_in_amo_languages(lang):
-    """Make sure all bidi marked locales are in AMO_LANGUAGES too."""
```

Comment:
I'm removing this test because that's no longer true, some bidi locales are no longer enabled on the site